core/Common.php

```diff
@@ -300,9 +300,8 @@ public static function safe_unserialize($string, $allowedClasses = [], $rethrow
             }
 
             $logger = StaticContainer::get('Psr\Log\LoggerInterface');
-            $logger->debug('Unable to unserialize a string: {message} (string = {string})', [
-                'message' => $e->getMessage(),
-                'backtrace' => $e->getTraceAsString(),
+            $logger->debug('Unable to unserialize a string: {exception} (string = {string})', [
+                'exception' => $e,
```

Comment:
Doesn't using the exception object here cause a cast to string when it's used in the message? And __toString will show the message and the (full) backtrace. Or does the Monologger do some magic here to use the ExceptionToTextProcessor?

Reply:
to my understanding, ExceptionToTextProcessor should convert the 'exception' value to a string